examples/postgres-operator/zarf.yaml

```diff
@@ -2,63 +2,55 @@ kind: ZarfPackageConfig
 metadata:
   name: postgres-operator-demo
   description: "Demo of prod-like Postgres database(s) on an edge cluster"
-  # Big Bang / Iron Bank are only amd64
-  architecture: amd64
+
+variables:
+  - name: PGADMIN_PASSWORD
+    prompt: true
 
 components:
-  - name: baseline
+  - name: postgres-operator
     required: true
-
-    manifests:
-      - name: postgres-example-config
-        files:
-          - manifests/patch-svc-accounts.yaml
-          - manifests/minio-instance-zarf-connect.yaml
-          - manifests/pgadmin-zarf-connect.yaml
-          - manifests/postgres-cluster.yaml
-          - manifests/postgres-operator.yaml
-          - manifests/postgres-operator-ui-zarf-connect.yaml
     charts:
       - name: postgres-operator
         url: https://opensource.zalando.com/postgres-operator/charts/postgres-operator
-        version: 1.7.0
+        version: 1.8.2
         namespace: postgres-operator
         valuesFiles:
-          - values/postgres-operator.yaml
+          - postgres/operator-values.yaml
       - name: postgres-operator-ui
         url: https://opensource.zalando.com/postgres-operator/charts/postgres-operator-ui
-        version: 1.7.0
+        version: 1.8.2
         namespace: postgres-operator
         valuesFiles:
-          - values/postgres-operator-ui.yaml
+          - postgres/operator-ui-values.yaml
+    manifests:
+      - name: zarf-connect
```

Comment:
The manifest later on in the file (line 46) shares the same manifest name.

When we generate a helm chart from a raw manifest internally we [currently](https://github.com/defenseunicorns/zarf/blob/3a6436c1cf03d4bbb3c76996167d03e9e136effe/src/internal/helm/chart.go#L176) use the manifest name as the chart name.
```tmpChart.Metadata.Name = fmt.Sprintf("raw-%s", manifest.Name)```



This means when the second component gets deployed, the raw manifest is turned into a helm chart that then upgrades the first `zarf-connect` manifest that got installed earlier, removing the service the other component installed. This screenshot shows the `zarf connect postgres-operator-ui` option is displayed the package is done being deployed since it did get installed originally but it's not actually available in the cluster because the `raw-zarf-connect` chart got upgraded immediately afterwards.


<img width="911" alt="Screen Shot 2022-08-15 at 10 01 45 PM" src="https://user-images.githubusercontent.com/18587823/184782821-e456d61e-a268-419b-8c91-f1c543f71ff0.png">

Reply:
We could add some validation logic to make sure this doesn't happen again within a single package but we should probably think about how to do this for different packages.

`zarf-connect` seems like a very reasonable manifest name (and I'm sure there are thousands names for other manifests that can be used often) for single package to have, but if someone were to deploy two independent packages who both had the reasonable `zarf-connect` name then one of the manifests would essentially get uninstalled.

At the same time, doing something like adding a random salt to the end of the chart names will make it harder to update zarf packages that have been deployed in the future.

Reply:
Would it make sense to scope the manifests to the component or package (something like `postgres-operator.zarf-connect`)

Reply:
That's a good point, doing something like that will probably be good enough. 

Since the chart name is something internal to the cluster and not something someone would see unless they were really looking for it, I would vote towards including the `{package}.{component}.{manifest}` as the chart name just to make it less likely to have an accidental collision.

Reply:
I forgot that the `releaseName` is [limited to 53 characters](https://helm.sh/docs/chart_template_guide/getting_started/#:~:text=TIP%3A%20The%20name%3A%20field%20is,are%20limited%20to%2053%20characters.) so maybe not to my last suggestion 🤦

Reply:
Yeah...that was the issue, it's actually super dumb...but I do have a terrible idea...